docs/source/quickstart.rst

```diff
@@ -3,6 +3,219 @@
+
+By default, MLFlow stores tracking data and artifacts in a ``./mlruns`` subdirectory of where you ran the code. You can change this behavior by setting the ``MLFLOW_TRACKING_URI`` environment variable to a different location. For instance, you can set it to a shared filesystem, a SQLAlchemy-compatible database, a tracking server, or a Databricks workspace.
+
+You can run a tracking server on a network-accessible server by running:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow server
+
+On your development machine, set the ``MLFLOW_TRACKING_URI`` environment variable to the URL of that server. For example, if your tracking server is listening on the default port of **5000** on the machine with IP address **192.168.0.1**, you can set the ``MLFLOW_TRACKING_URI`` environment variable to:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        export MLFLOW_TRACKING_URI=http://192.168.0.1:5000
+
+Now, when you run your code, it will send tracking data to the tracking server. To view the results, run the tracking UI on your local machine and point it to the tracking server:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui --backend-store-uri http://192.168.0.1:5000
+
+..
+
+    CONTENT REVIEW: Maybe that's all we need for *this* page? Short and sweet, you've: 
+
+    - installed MLFlow
+    - used `autolog` and explicit logging
+    - run the tracking UI
+    - run the tracking server
+
+    Maybe we end there and "Next Steps" it? 
+
+    If not:
+
+How do I use MLFlow with a Databricks workspace?
+------------------------------------------------
+
+tk 
+
+
+How do I store a model in MLFlow?
+---------------------------------
+
+tk 
+
+
+How do I run a model artifact from a specific MLFlow run?
+---------------------------------------------------------
+
+tk
+
+How do I do a hyperparameter sweep with MLFlow Tracking?
+--------------------------------------------------------
+
+tk
+
+Next Steps
+----------
+.. 
+    First, code:
+
+- Example: MLFlow tutorial
+- MLFlow example directory
+
+.. 
+    Drilldown for each H2. These could also be at the bottom
+    of their respective sections. Depends on info architecture design.
+
+- Customizing and troubleshooting MLFlow installation
+- Using Autolog, including a list of what libraries are supported
+- Effectively using the MLFlow Tracking API
+- Choosing and configuring the MLFlow tracking backend 
+- Navigating the MLFlow Tracking UI
+
+.. 
+    Next tool: either Models or Projects
+
+- Storing models with MLFlow tracking
+- End-to-end tracking, collaboration, and deployment with MLFlow Projects
+
+.. 
+    More top-down-y stuff
+
+- Understanding MLFlow's concepts, tools, and APIs 
+- Why MLFlow?
+  
+
+{>> Important things in original we don't want to lose:
+
+- mlflow skinny and extras and why you might choose one or t'other
+- Do we document / walkthrough virtualenvs or, e.g., conda or is that all assumed?
+- What about R? 
```

Comment:
Just calling out that the library can be installed in Java / R is plenty here. R isn't widely used.

Reply:
Done in the drill down. I actually ended up linking over to the R API page, since there's an intro page. For Java, I "due to the complexity of the Java ecosystem..." bailed.